INSTALL.md

```diff
@@ -114,30 +133,44 @@ index 1c4cc0e5b..2201bc43c 100644
        args << "--without-mysql"
 ```
 
-You can also use `brew edit sphinx` and apply the modified lines
+Run `brew edit sphinx` to edit Sphinx configuration or alternatively open `${HOMEBREW_PREFIX:-/usr/local}/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/sphinx.rb` with your favorite editor.
+
+### Setup
+
+#### Eventmachine
 
-Then install sphinx with mysql@5.7 version
+Eventmachine has to be installed with `--with-cppflags=-I/usr/local/opt/openssl/include`. Simply run:
 
 ```shell
-brew install sphinx --with-mysql@5.7
+bundle config build.eventmachine --with-cppflags=-I/usr/local/opt/openssl/include
 ```
 
+#### Config files
 
-### Pre setup
-
-Eventmachine should be installed with `--with-cppflags=-I/usr/local/opt/openssl/include`.
+Copy example config files from the examples folder:
 
 ```shell
-bundle config build.eventmachine --with-cppflags=-I/usr/local/opt/openssl/include
+cp config/examples/* config/
 ```
 
-### Setup
+#### Bundle
 
-Copy examples config:
+Run [Bundler](https://bundler.io/) to install all required Ruby gems:
 
 ```shell
-cd porta
-cp config/examples/* config/
 bundle install
+```
+
+#### Setup Database
+
+Finally initialize the database with some seed data by running:
+
+```bash
 bundle exec rake db:setup
 ```
+
+You may need to set the database up from scratch again, in that case use `db:restart` to drop it first too:
+
+```bash
+bundle exec rake db:restart # This will drop and setup the database
```

Comment:
There is no such task I think

Reply:
I think he meant `db:reset` which does something like `bundle exec rake db:drop db:create db:migrate`. Although I guess for us it would be better `bundle exec rake db:drop db:create db:setup` if they already have the DB, and just `bundle exec rake db:setup` if they start from scratch. We need the `seeds`, and it is better if they load the schema than if they try to run our migrations (btw, they will fail if the DB doesn't contain data... 😓 )